primitives/runtime/src/traits.rs

```diff
@@ -84,10 +84,18 @@ pub trait Verify {
 	type Signer: IdentifyAccount;
 	/// Verify a signature.
 	///
-	/// Return `true` if signature is valid for the value.
+	/// Should return `true` if signature is valid for the value.
 	fn verify<L: Lazy<[u8]>>(&self, msg: L, signer: &<Self::Signer as IdentifyAccount>::AccountId) -> bool;
 }
 
+/// Means of signature batch verification.
```

Comment:
This should have better documentation. Saying that you first need to start batching etc and that you need to finish batching.

Reply:
Yeah, but probably not here

This is just a trait, and you describe implementation.

Reply:
Please describe it here. The user of the trait needs to be aware of this.